src/internal/CircleRipple.js

```diff
@@ -55,20 +58,47 @@ class CircleRipple extends React.Component {
   }
 
   animate() {
-    const style = ReactDOM.findDOMNode(this).style;
-    const transitionValue = `${transitions.easeOut('2s', 'opacity')}, ${
-      transitions.easeOut('1s', 'transform')}`;
-    autoPrefix.set(style, 'transition', transitionValue);
-    autoPrefix.set(style, 'transform', 'scale(1)');
+    // These animations are driven with CSS animations because CSS transitions
+    // don't provide a start even
+    const node = ReactDOM.findDOMNode(this);
+    // TODO: Support prefixing for reals (I'll do this before landing the change)
+    node.addEventListener('webkitAnimationStart', this.props.onRippleStart);
```

Comment:
Do we really need to wait for the CSS animation to start?
I'm wondering if waiting for the asynchronous `componentDidEnter` / `componentDidAppear` is enough.
Having to use an animation event callback scares me.
 - We need to handle correctly the prefixing, but our current inline-style approach make is difficult
 - I'm wondering if those events are reliable as discussed here https://github.com/facebook/react/issues/1326. They basically stopped using it.